deeprank/learn/NeuralNet.py

```diff
@@ -1145,12 +1160,12 @@ def _get_relevance(self, data, irmsd_thr=4.0):
         # get the target values
         out = data['outputs']
 
-        # get the irmsd
-        irmsd = []
+        # get the targets
+        targets = []
         for fname, mol in data['mol']:
 
             f5 = h5py.File(fname, 'r')
-            irmsd.append(f5[mol + '/targets/IRMSD'][()])
+            targets.append(f5[mol + f'/targets/self.data_set.select_target'][()])
```

Comment:
need to change `f'/targets/self.data_set.select_target'` to `f'/targets/{self.data_set.select_target}'`